include/onnxruntime/core/session/onnxruntime_lite_custom_op.h

```diff
@@ -619,12 +869,14 @@
 template <typename... Args>
 struct OrtLiteCustomFunc : public OrtLiteCustomOp {
   using ComputeFn = void (*)(Args...);
+  using ComputeFnV2 = Status (*)(Args...);
```

Comment:
>ComputeFnV2 [](http://example.com/codeflow?start=8&length=11)

V2 sounds too strong... here we just provide an overload to support return status, right? maybe "ComputeFnWithStatus"?

Reply:
How about "ComputeFnReturnStatus"?